R/compute.R

```diff
@@ -265,7 +269,14 @@ create_aks_compute <- function(workspace,
                                subnet_name = NULL,
                                service_cidr = NULL,
                                dns_service_ip = NULL,
-                               docker_bridge_cidr = NULL) {
+                               docker_bridge_cidr = NULL,
+                               cluster_purpose = NULL) {
+  
+  if (!is.element(cluster_purpose, c('DevTest', 'FastProd'))) {
+    stop(paste(cluster_purpose, "is not a valid cluster_purpose value.
+               Please see documentation."))
+  }
+
```

Comment:
then we can just do `match.arg(cluster_purpose)` here, which will automatically give an error message if the input doesn't match the valid values

Reply:
great point - i've just refactored it